examples/generative/corrdiff/training/training_loop.py

```diff
@@ -154,6 +157,8 @@ def training_loop(
     )
 
     img_in_channels = len(in_channels)  # noise + low-res input
+    if img_shape_x!=patch_shape_x or img_shape_y!=patch_shape_y:
+        img_in_channels *= 2
```

Comment:
This is not clear to me.Would be great if you can please add a comment next to this and explain it